classes/db/DbPDO.php

```diff
@@ -60,6 +60,7 @@ protected static function _getPDO($host, $user, $password, $dbname, $timeout = 5
         } else {
             $dsn .= 'host='.$host;
         }
+        $dsn .= ';charset=utf8';
```

Comment:
I can't ask you to modify anything because it was already hardcorded in the previous version, but it seems the charset defined in `app/config/parameters.yml` is only taken in account in the doctrine / DBAL connections, not here.